Assets/LeapMotion/Scripts/Utils/Utils.cs

```diff
@@ -167,6 +167,34 @@ public static class Utils {
 
     #endregion
 
+    #region Collider Utils
+
+    public static Vector3 GetDirection(this CapsuleCollider capsule) {
+      return capsule.direction == 0 ? Vector3.right : capsule.direction == 1 ? Vector3.up : Vector3.forward;
+    }
+
+    /// <summary>
+    /// Manipulates capsule.transform.position, capsule.transform.rotation, and capsule.height
+    /// so that the line segment defined by the capsule connects world-space points a and b.
+    /// </summary>
+    public static void SetCapsulePoints(this CapsuleCollider capsule, Vector3 a, Vector3 b) {
+      capsule.center = Vector3.zero;
+
+      capsule.transform.position = (a + b) / 2F;
+
+      Vector3 capsuleDirection = capsule.GetDirection();
+
+      Vector3 capsuleDirWorldSpace = capsule.transform.TransformDirection(capsuleDirection);
+      Quaternion necessaryRotation = Quaternion.FromToRotation(capsuleDirWorldSpace, a - capsule.transform.position);
+      capsule.transform.rotation = necessaryRotation * capsule.transform.rotation;
+
+      Vector3 aWorldSpace = capsule.transform.InverseTransformPoint(a);
+      float capsuleSpaceDistToA = aWorldSpace.magnitude;
```

Comment:
Ooooh i see what's going on, aWorldSpace is not in world space, but it's not supposed to be.  You are getting the transform local distance from the position to a, makes sense!

So in that case my only suggested change here would be to rename aWorldSpace to aLocalSpace or aCapsuleSpace.